packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -47,6 +48,24 @@ systemd:
         [Install]
         RequiredBy=kubelet.service
         RequiredBy=etcd-member.service
+    - name: create-etcd-private-ip-env-file.service
```

Comment:
Maybe rename `create-etcd-private-ip-env-file.service` to `create-etcd-config.service` since the file name "etcd.config" is generic?

Reply:
done